templates/app/client/components/auth(auth)/auth.service.js

```diff
@@ -100,7 +100,7 @@ export function AuthService($location, $http, $cookies, $q, appConfig, Util, Use
     /**
      * Gets all available info on a user
      *
-     * @param  {Function} [callback] - funciton(user)
+     * @param  {Function} [callback] - optional, function(user)
      * @return {Promise}
      */
     getCurrentUser(callback) {
```

Comment:
Yeah that's a more informative type definition for it. Was not sure it was valid Flow syntax as well but it does appear to be. Examples or documentation of each for reference: [Flow documentation](https://flowtype.org/docs/functions.html) [TS example](http://www.typescriptlang.org/play/#src=export%20interface%20INotificationService%20%7B%0A%20%20%20%20error(message%3A%20string%2C%20title%3F%3A%20string%2C%20autoHideAfter%3F%20%3A%20number)%3B%0A%7D%0A%0Aclass%20X%20%7B%0A%09error(message%3A%20string%2C%20title%3F%3A%20string%2C%20autoHideAfter%3F%3A%20number)%20%7B%0A%09%09console.log(message%2C%20title%2C%20autoHideAfter)%3B%0A%09%7D%0A%7D%0A%0Anew%20X().error(%22hi%20there%22%2C%20undefined%2C%201000)%3B).

Should probably change all the other async functions then in auth.service as I kinda only copied documentation from them while all of them use the same `Util.safeCb` to make it optional. 

Will just do it now.